src/crab/ebpf_checker.cpp

```
@@ -213,29 +218,29 @@ void EbpfChecker::operator()(const ValidCallbackTarget& s) const {
     }
 
     const int32_t callback_label = callback_target->cast_to<int32_t>();
-    if (!thread_local_program_info->callback_target_labels.contains(callback_label)) {
+    if (!context.program_info.callback_target_labels.contains(callback_label)) {
         throw_fail("callback function pointer does not reference a valid callback entry");
     }
```

Cppcheck (2.20.0)

[style] The function 'hash_value' is never used.

(unusedFunction)